Vision/Vision.bs

```diff
@@ -142,34 +143,34 @@ Boilerplate: omit conformance
 		and more.
 		In order to ensure the W3C serves the needs of the entire Web user base, 
 		we also strive to broaden diversity and inclusion for our own participants.
-	* We create standards with an ethical intent to improve 
+	* **Horizontal Review**: We create standards with an ethical intent to improve 
 		equity of access and participation on the Web. 
```

Comment:
Access Equity is a better summary and is in the text

Reply:
This needs bikeshedding. we should do that in https://github.com/w3c/AB-public/pull/122/files after this lands.